localenv/mock-account-servicing-entity/app/routes/rates.ts

```diff
@@ -7,16 +7,16 @@ interface Rates {
 
 const exchangeRates: Rates = {
   USD: {
-    EUR: 1.1602,
-    ZAR: 17.3792
+    EUR: 0.89,
```

Comment:
now, the rates that the external API needs to return are more intuitive: 1 USD = 0.89 EUR ✅